openpype/hosts/houdini/api/lib.py

```diff
@@ -562,19 +562,41 @@ def get_frame_data(node):
         dict: frame data for star, end and steps.
 
     """
+    if asset_data is None:
+        asset_data = {}
+
+    if log is None:
+        log = self.log
+
     data = {}
 
     if node.parm("trange") is None:
-
+        log.debug(
+            "Node has no 'trange' parameter: {}".format(node.path())
+        )
         return data
 
     if node.evalParm("trange") == 0:
-        self.log.debug("trange is 0")
-        return data
-
-    data["frameStart"] = node.evalParm("f1")
-    data["frameEnd"] = node.evalParm("f2")
-    data["steps"] = node.evalParm("f3")
+        data["frameStartHandle"] = hou.intFrame()
+        data["frameEndHandle"] = hou.intFrame()
+        data["byFrameStep"] = 1.0
+        data["handleStart"] = 0
+        data["handleEnd"] = 0
+        log.info(
+            "Node '{}' has 'Render current frame' set. \n"
+            "Asset Handles are ignored. \n"
+            "frameStart and frameEnd are set to the "
+            "current frame".format(node.path())
+        )
+    else:
+        data["frameStartHandle"] = node.evalParm("f1")
```

Comment:
can we make sure this is always an int? I have gotten some publishes out of Houdini where frameStart/End were floats and that was causing errors in Nuke loader

Reply:
yeah, sure!